modules/sdk/project-templates-war-hook/src/main/resources/archetype-resources/src/main/java/__className__StartupAction.java

```diff
@@ -0,0 +1,25 @@
+package ${package};
+
+import com.liferay.portal.kernel.events.ActionException;
+import com.liferay.portal.kernel.events.SimpleAction;
+
+import java.util.stream.Stream;
+
+/**
+ * @author ${author}
+ */
+public class ${className}StartupAction extends SimpleAction {
+
+	@Override
+	public void run(String[] lifecycleEventIds) throws ActionException {
+		Stream<String> lifecycleEventIdsStream = Stream.of(lifecycleEventIds);
+
+		lifecycleEventIdsStream.map(
+			id -> "startup event id " + id
+		).forEach(
+			System.out::println
+		);
+
```

Comment:
@gamerson and @lawrence-lee, why not writing something like this instead?
```java
for (String eventId : lifecycleEventIds) {
	System.out.println("startup event id " + eventId);
}
```
It's 3 lines instead of 7 😅